SmartDeviceLink/private/SDLIAPTransport.m

```diff
@@ -92,7 +96,13 @@ - (void)sdl_stopEventListening {
  */
 - (void)sdl_accessoryConnected:(NSNotification *)notification {
     EAAccessory *newAccessory = notification.userInfo[EAAccessoryKey];
-
+    double timeNow = NSTimeIntervalSince1970;
+    double timeSinceLastConnectionAttempt = timeNow - _eaAccessoryConnectedTime;
+    if (timeSinceLastConnectionAttempt < TimeRequiredBetweenConnectionAttempts) {
+        SDLLogD(@"Accessory Connecting too soon after previous attempt");
+        return;
+    }
+    _eaAccessoryConnectedTime = timeNow;
```

Comment:
I can't remember any rationale for this change. Ignoring a notification can be very dangerous leading to no app connecting. Can you give us a reason why this code is needed?